src/shared/sleep-config.c

```diff
@@ -193,25 +189,14 @@ static int read_battery_capacity_percentage(sd_device *dev) {
 int battery_is_low(void) {
         _cleanup_(sd_device_enumerator_unrefp) sd_device_enumerator *e = NULL;
         sd_device *dev;
-        int r;
 
          /* We have not used battery capacity_level since value is set to full
          * or Normal in case acpi is not working properly. In case of no battery
          * 0 will be returned and system will be suspended for 1st cycle then hibernated */
 
-        r = battery_enumerator_new(&e);
```

Comment:
why is the initialization being removed here?

Reply:
Ouch. Fixed.